Casbin/Model/DefaultModel.cs

```diff
@@ -262,6 +262,74 @@ public void SortPoliciesByPriority()
             }
         }
 
+        public void SortPoliciesBySubjectHierarchy()
+        {
+            if (!Sections["e"]["e"].Value.Equals("subjectPriority(p.eft) || deny"))
+            {
+                return;
+            }
+            var subjectHierarchyMap = GetSubjectHierarchyMap(Sections["g"]["g"].Policy);
+            foreach (var keyValuePair in Sections["p"])
+            {
+                var ptype = keyValuePair.Key;
+                var assertion = keyValuePair.Value;
+                assertion.TrySortPoliciesBySubjectHierarchy("dom", subjectHierarchyMap, GetNameWithDomain);
+                Console.WriteLine();
+            }
+        }
+
+        private string GetNameWithDomain(string domain, string name)
+        {
+            return domain + "::" + name;
```

Comment:
Is there any way to avoid using a default Separator `::`?

Reply:
Maybe we could create a new class for it to describe the ```NameWithDomain```?